### PR TITLE
title_wrong_on_converted_repo

### PR DESCRIPTION
**Pull request by bspidel**
add resource types to resource map

Issue-759

### CODE REVIEW

File: libraries/resource_container/ResourceContainer.py [hunk 1]
```diff
@@ -35,6 +35,16 @@
         'type': 'book',
         'format': 'text/markdown'
     },
+    'obs-tn': {
+        'title': 'OBS TranslationNotes',
```
Comment (richmahn) [inline libraries/resource_container/ResourceContainer.py]: Lowercase the `T` of `Translation`

File: libraries/resource_container/ResourceContainer.py [hunk 2]
```diff
@@ -35,6 +35,16 @@
...
+        'type': 'help',
+        'format': 'text/markdown'
+    },
+    'obs-tq': {
+        'title': 'OBS TranslationQuestions',
```
Comment (richmahn) [inline libraries/resource_container/ResourceContainer.py]: Lowercase the `T` of `Translation`

**Review by richmahn — CHANGES_REQUESTED** [review]
Lowercase the `T` of `Translation`

**Comment by bspidel** [comment]
@richmahn Made case change you requested

**Review by richmahn — APPROVED** [review]
LGTM

**Pull request merged**